src/lib/features/scheduler/scheduler-service.ts

```diff
@@ -93,9 +93,11 @@ export class SchedulerService {
 
             if (!maintenanceMode) {
                 if (jitter) {
-                    setTimeout(() => {
-                        runScheduledFunctionWithEvent();
-                    }, jitter);
+                    const id = setTimeout(
+                        () => runScheduledFunctionWithEvent(),
+                        jitter,
+                    );
+                    this.intervalIds.push(id);
```

Comment:
just to make sure: can clearInterval clear setTimeout?

Reply:
Good question. we should test.

Reply:
We're good here.
`const x2 = setTimeout(() => console.log("Asdf"), 1000); clearInterval(x2);`
works in browser and node. https://developer.mozilla.org/en-US/docs/Web/API/clearInterval

Reply:
It will work with the current node.js version. But this might break in the future, and we should have a separate list for setTimeout timers.